pynio/block.py

```diff
@@ -1,3 +1,4 @@
+import pprint
```

Comment:
If you're going to use pprint, it needs to be added to setup.py. I would prefer that it didn't need to add this dependency though. Wouldn't printing the name of the Block/Service be a good enough __str__?